src/Knp/Snappy/AbstractGenerator.php

```diff
@@ -380,7 +380,7 @@ abstract protected function configure();
      *
      * @return $this
      */
-    protected function addOption($name, $default = null)
+    protected function addOption($name, ?$default = null)
```

Comment:
This change needs to be reverted.

Reply:
I do not understand. Why does it need to be reverted?

Reply:
Nevermind, I rectified the error. Typical gnito fault.